### PR TITLE
Use typings; tsd is deprecated

### PR DESCRIPTION
**Pull request by mikekidder**
Was going to push earlier, but saw you had updated tsd.  :)

package.json - removed tsd, adds typings.... the rest is just from alpha sorting

### CODE REVIEW

**Comment by mikekidder** [comment]
CLA, I signed it!

File: package.json [hunk 1]
```diff
@@ -3,7 +3,6 @@
   "version": "0.5.14",
   "description": "Zones for JavaScript",
   "main": "build/lib/zone.js",
-  "typings": "build/lib/core.d.ts",
```
Comment (alexeagle) [inline package.json]: Don't remove this line - this is an output of the package. It allows users to
`import {Zone} from 'zone.js';`
and the typescript type-checker can find it.

**Comment by alexeagle** [comment]
LGTM after that fix

File: package.json (hunk 1 above)
Comment (mikekidder) [inline package.json]: Ah, good to know. Wasn't sure if that was a tsd leftover.

**Comment by mikekidder** [comment]
Closing as this looks to be covered in 0.6.1

**Pull request closed without merging**